testing/integration/helpers.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
+
+package integration
```

Comment:
The filename should include endpoint or just be endpoint.go or something. I don't think you want this file to be a generic dumping ground for utility functions.